external/CMakeLists.txt

```diff
@@ -78,7 +78,8 @@ if(WITH_HIVE)
   add_subdirectory(pybind11 EXCLUDE_FROM_ALL)
 endif()
 
-add_subdirectory(cxxopts EXCLUDE_FROM_ALL)
+# replacing cxxopts with CLI11
```

Comment:
Comment seems like a temporary comment to be removed

Reply:
fixed